### PR TITLE
a simple makefile for the C version

### PR DESCRIPTION
**Pull request by gritzko**
Bazel is a big overshoot for such a compact project.
I link ryu through CMake's ExternalProject, so having a Makefile simplifies things a lot.

### CODE REVIEW

**Comment by ulfjack** [comment]
Except the Makefile here doesn't work on all platforms (e.g., Windows). When I ported Ryu to Mac and Windows, I didn't really have to modify the BUILD files, and it mostly just worked out of the box with Bazel.

I acknowledge that Bazel is a bit heavyweight, and requires an additional install that people would rather not have to go through. As such, I'm willing to have the Makefile be added here. However, I can't guarantee that it'll continue to work.

Please add a disclaimer to the top of the Makefile that this is a contributed file, and that it's maintained on a best-effort basis.

**Comment by Finii** [comment]
Adding at least the tests should not be too much work, right? And if you use this in another project you want the tests to be incorporated, no? _Compiles on my machine_ is of course also an approach.
:smiley_cat:

**Comment by gritzko** [comment]
> Adding at least the tests should not be too much work, right?

That broadens the scope. The is a gtest dependency. My project has my own copy of gtest, there might or might not be system-default gtest... That increases the surface area. 
If the dependency version is fixed by hash, that might also be redundant.

**Comment by ulfjack** [comment]
The Makefile can contain rules to build as well as test it, and just not work for testing if gtest isn't there. Just sayin'. I'm happy to merge this without test support.

**Comment by ulfjack** [comment]
To be clear: I'm happy to merge this if you add a comment that this is contributed and support is best effort. It's ok *not* to have support for multiple platforms, or tests, for now.

File: ryu/Makefile [hunk 1]
```diff
@@ -37,4 +34,20 @@ uninstall:
 	rm -f $(DESTDIR)$(PREFIX)/lib/$(ALIB)
 	rm -f $(DESTDIR)$(PREFIX)/include/ryu.h
 
+TESTSRC=tests/d2fixed_test.cc tests/d2s_intrinsics_test.cc tests/d2s_table_test.cc tests/d2s_test.cc tests/f2s_test.cc tests/generic_128_test.cc
+
+TESTS = $(TESTSRC:.cc=.test)
+
+# Assumes gtest is somehow available (e.g. installed in the system)
```
Comment (Finii) [inline ryu/Makefile]: `gtest` is within this repository... You would just need to make it ;-)

Just an observation, I don't want to trigger anything here.

**Pull request merged**